### ISSUE TITLE
Talkback related issues for landscape 

### ISSUE THREAD

**Issue by JuliannaR** [cds-snc/covid-alert-app#704]
Focus goes to top of the page after going through "Learn more about how it works" screens AND "The app will now ask your permission" screen
https://github.com/cds-snc/covid-shield-mobile/issues/689

Focus doesn't go to heading in landscape orientation #675
https://github.com/cds-snc/covid-shield-mobile/issues/675

**Comment by timarney** [comment]
@nickmly please give this a look based on the PRs from last night.

**Comment by nickmly** [comment]
@timarney I can confirm that #675 is fixed as of last night, but #689 is still an issue. #825 will fix this

**Comment by nickmly** [comment]
@JuliannaR  @timarney 

Closing #825 in favor of #834. However the accessibility issue in Step 3 and Step 5 of onboarding still exists. (#689)

This is due to the button for these modals being at the bottom of the screen, so when returning from the modal TalkBack cannot read the 'Step x of 6' text as it is off screen. A potential solution is to scroll back to the top when the user returns from the modal.

cc. @henrytao-me

**Comment by timarney** [comment]
Closing this - should be retested.
